litestar/datastructures/secret_values.py

```diff
@@ -0,0 +1,120 @@
+from typing import Any, Union
```

Comment:
We can replace with `from __future__ import annotations` to relieve some future maintenance burden here?

And replace subsequent usages of Union e.g., `secret_value: str | bytes`

Reply:
Done